### ISSUE TITLE
Add iGARCH as an option for enforcing alpha+beta=1 or not

### ISSUE THREAD

**Issue by EchoRLiu** [EchoRLiu/robustGarch#50]
See #45 for the discussion

**Comment by EchoRLiu** [comment]
Currently, only non-`MLE` (or `tdistMLE ` after #51) will be changed.

Which actually raises the question: isn't iGARCH only possible for QML? so shouldn't iGARCH not allowed for other methods? Maybe this can be a separate issue to address to enforce that?

**Comment by EchoRLiu** [comment]
another question is whether we should print this when using the `print()`?

at least for the QML case?
